apps/backend/src/mcp/resources/proxy_configs.py

```
@@ -0,0 +1,1071 @@
+                        'description': f"SWAG sample configuration for {sample['service_name']} ({sample['sample_type']})",
+                        'mime_type': 'text/plain'
+                    })
+        except Exception as e:
+            logger.error(f"Error discovering sample files: {e}")
+        
+        # Add individual active service configuration resources by discovering them
+        try:
+            swag_device = "squirts"
+            # Discover all active .subdomain.conf and .subfolder.conf files
+            ls_command = """
+            find /mnt/appdata/swag/nginx/proxy-confs -name '*.subdomain.conf' -o -name '*.subfolder.conf' | grep -v '\.sample$' | sort
```

Ruff (0.12.2)

Invalid escape sequence: `\.`

Use a raw string literal

(W605)